src/applications/personalization/dashboard/utils/timezone.js

```diff
@@ -104,6 +104,14 @@ export function getUserTimezoneAbbr() {
  *   - description: The written out description (e.g. Eastern time)
  */
 export function getAppointmentTimezone(appointment) {
+  if (appointment?.timeZone) {
+    return {
+      identifier: appointment.timeZone,
+      abbreviation: appointment.timeZone,
```

Comment:
Should this use the result `splitDST()` similarly to line 132, so:

```
    const abbreviation = stripDST(
      appointment.timeZone?.split(' ')?.[1] || appointment.timeZone,
    );
```
Or is this abbreviation always formatted correct?

Otherwise lgtm

Reply:
It's meant for v1. V1 has a timezone attribute that is just a string abbreviation